### PR TITLE
Best position for a service centre

### LINKED ISSUE #9: Service Center
https://leetcode.com/problems/best-position-for-a-service-centre/description/

### PR DESCRIPTION
**Pull request by begoat**
Close #9

### CODE REVIEW

**Comment by begoat** [comment]
Will take a look at others' solutions.

**Comment by begoat** [comment]
https://www.youtube.com/watch?v=8IX9j5WLLD4
I use strategy described in this video. Just move point and test step by step.

File: ao/test_best_position_for_a_service_centre.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    def test_eg3(self):
+        self.assert_equal_with_precision(SOLUTION.getMinDistSum([[1,1]]), 0.00000)
+
+    def test_eg4(self):
+        self.assert_equal_with_precision(SOLUTION.getMinDistSum([[1,1],[0,0],[2,0]]),  2.73205)
+
+    def test_eg5(self):
+        self.assert_equal_with_precision(SOLUTION.getMinDistSum([[0,1],[3,2],[4,5],[7,6],[8,9],[11,1],[2,12]]), 32.94036)
+
+
```
Comment (zzj0402) [inline ao/test_best_position_for_a_service_centre.py:33]: [[4,4],[52,89],[76,60],[4,4],[4,4],[93,59],[50,92],[4,4],[76,14],[4,4],[46,41],[4,4],[4,4],[4,4],[4,4],[67,14],[73,71],[83,44],[4,4],[4,4],[4,4],[4,4],[30,29],[74,77],[4,4],[4,4],[4,4],[26,62],[4,4],[4,4],[50,30],[44,93]]

answer expected: 1119.58582

answer of your code: 
1119.58589

add test case

**Review by zzj0402 — CHANGES_REQUESTED** [review]
add more test case

File: ao/test_best_position_for_a_service_centre.py (hunk 1 above)
Comment (begoat) [reply ao/test_best_position_for_a_service_centre.py:33]: I have passed the leetcode testcase :)

Where are your testcase from ? LOL
Actually your testcase is really helpful.

**Pull request merged**